services/workflows-service/src/alert/alert.service.ts

```diff
@@ -340,4 +403,25 @@ export class AlertService {
       [projectId],
     );
   }
+
+  sortBySeverity(a: AlertSeverity, b: AlertSeverity) {
```

Comment:
Why do we need it? we have sort already in prisma just add an `orderby` clause section